### ISSUE TITLE
IM-Net and Occ-Net

### ISSUE THREAD

**Issue by Vinayak-VG** [czq142857/IM-NET#15]
Hey, could you please tell me the difference between IM-Net and Occupancy Networks. Both look exactly the same to me. 
It would be really helpful if you help me out with this question

**Comment by czq142857** [comment]
There are a few differences in network designs and loss function. But other than that, they are almost the same. In fact, there are three papers proposing the same idea (continuous implicit representation) in CVPR 2019, pretty amazing. I encourage you to take a look at all of them:

DeepSDF and IM-NET: These networks take a shape feature code and point coordinates as inputs, and pass them to several fully-connected layers, possibly with skip connections.

OccNet: Occupancy networks use the shape feature code to predict the parameters for conditional batch normalization in the fully-connected layers, and adopt ResNet blocks in their structures.

**Comment by Vinayak-VG** [comment]
Thanks a lot for the information
It really helped me a lot :)
